src/frameworks/opengles/gles1_on_gl2.rs

```diff
@@ -665,6 +665,10 @@ impl GLES for GLES1OnGL2 {
         );
         gl21::TexParameteri(target, pname, param);
     }
+    unsafe fn TexParameterf(&mut self, target: GLenum, pname: GLenum, param: GLfloat) {
+        assert!(target == gl21::TEXTURE_2D);
```

Comment:
Huh, what parameter is the app trying to set? Looking at https://registry.khronos.org/OpenGL-Refpages/es1.1/xhtml/ I think all the parameters are integers, so maybe it's relying on the OpenGL implementation doing a float-to-integer conversion. If so, you can copy the assert from `TexParameteri`, but add a comment that they are all integers.

Reply:
GL_TEXTURE_MAX_ANISOTROPY_EXT, which is part of the anisotropic filtering extension to OpenGL ES 1. Apparently, iPhone OS has supported it since 2.0: https://developer.apple.com/documentation/opengles/gl_texture_max_anisotropy_ext?language=objc

Reply:
So it looks like the OpenGL bindings don't have constants for either of these. What would be the correct way to add them?

Reply:
Have a look at `src/gl_bindings/build.rs`. It has a list of extensions to include in the generated bindings.